docs/README.md

```diff
@@ -20,12 +20,24 @@ Temporal fixes these problems by:
 - Parsing a strictly specified string format
 - Supporting non-Gregorian calendars
 
+Temporal provides a number of separate types, in order to
+
+- Represent the information you have
+- Avoid buggy pattern of filling in 0 or UTC for missing info
+- Do appropriate calculations based on type
+
+
 ## Cookbook
 
 A cookbook to help you get started and learn the ins and outs of Temporal is available [here](./cookbook.md).
 
 ## API Documentation
 
+Types whose name starts with "Plain" (like `Temporal.PlainDate`, `Temporal.PlainTime`, and `Temporal.PlainDateTime`) do not, by convention, have an associated time zone.
```

Comment:
"By convention" sticks out strangely here (as if it's a convention that Temporal users should follow, as opposed to one that the library follows). I would word this as,

> The Temporal API follows a convention of using types whose names start with "Plain" "Plain" (like `Temporal.PlainDate`, `Temporal.PlainTime`, and `Temporal.PlainDateTime`) for objects which do not associated time zone.